scout/parse/variant/clnsig.py

```diff
@@ -100,20 +100,23 @@ def is_pathogenic(variant):
         bool: If variant should be loaded based on clinvar or not
     """
 
-    load_categories = {
-        "pathogenic",
-        "likely_pathogenic",
-        "conflicting_interpretations_of_pathogenicity",
-        "conflicting_interpretations",
-    }
+    quick_load_categories = {"pathogenic", "conflicting_"}
 
     # check if VEP-annotated field contains clinvar pathogenicity info
     vep_info = variant.INFO.get("CSQ")
     if vep_info:
-        for category in load_categories:
+        for category in quick_load_categories:
             if category in vep_info.lower():
                 return True
 
+    load_categories = {
+        "pathogenic",
+        "likely_pathogenic",
+        "conflicting_classifications_of_pathogenicity",
+        "conflicting_interpretations_of_pathogenicity",
+        "conflicting_interpretations",
+    }
+
     # Otherwise check if clinvar pathogenicity status is in INFO field
     clnsig_accessions = parse_clnsig(variant)
```

Comment:
It could help to add a type hint to the variable

Reply:
Eh, that should always be apparent on declaration, I hope? You mean one could mistake it for a dict? Why not, yes - it's not so commonly used I guess. Or in the def statement? I'll fix that as well to be sure.

Reply:
Ah, I hear you! I'll add a return type to `parse_clnsig` instead - should solve it for most editors!